101-load-balancer-standard-create/README.md

```diff
@@ -0,0 +1,20 @@
+# Create an Internet-facing standard load-balancer with three VMs
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-load-balancer-standard-create%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-load-balancer-standard-create%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+**This template provides an example for how to deploy a standard load balancer.**
+
+The template creates and configures the following Azure resources:
+
+- a load balancer
+- a network security group
+- a virtual network
+- a virtual network interface
```

Comment:
Please replace with - three NICs associated with the backend pool of the load balancer